virttest/env_process.py

```diff
@@ -1167,8 +1167,32 @@ def preprocess(test, params, env):
     if (params.get("auto_cpu_model") == "yes" and
             vm_type == "qemu"):
         if not env.get("cpu_model"):
```

Comment:
Seems env is not cleared when case finishes. If the next case runs with different machine type, the cpu info stored in env might be incorrect. Shall we remove this if condition here?

Reply:
Not only when machine type change, the next case might use different policies, then the env['cpu_*'] is definitely not what it needs. So I remove it. Thanks.